R/WIS_density_plot.R

```diff
@@ -0,0 +1,57 @@
+#' Title
+#'
+#' @param wis_data
+#'
+#' @return
+#' @export
+#'
+#' @examples
+
+
+wis_density_plot <- function(wis_data){
+
+  require(dplyr)
+  require(ggplot2)
+  require(ggridges)
```

Comment:
I think it's good practive to avoid `require()` in a function. `require()` try to load a package and do not stop if the package is missing, it just returns `FALSE` and a warning message if the package is not present. 

A good way to be certain to have the package of interest loaded with your package is to added it to your dependencies. 

For that, you can use 2 options:

**Option 1:** the `importFrom` tag from roxygen2:
```
@importFrom dplyr group_by mutate %>% n row_number arrange ungroup
@importFrom stats reorder quantile
...
```
You can add it around your `@export` tag and it should be one line per package with:
`@importFrom` followed by the name of the package and the list of functions you need are using, separated by a space. 
You only need to import a function once, no matter how many times you use @importFrom foo bar you’ll only get one importFrom(foo, bar).

Then you can run:
```
devtools::document()
```
and it will automatically about the NAMESPACE file (record of the all the dependencies).


**Option2:** for each function issued of a specific package,
indicate from which package the function is coming, by using the `::`.

For example:
```
inc_scores_overall <- WIS_data %>%
        dplyr::group_by(target_end_date, target, location_name) %>%
        dplyr::mutate(n_models = n()) %>%
        dplyr::arrange(WIS) %>%
        dplyr::mutate(model_rank = row_number(), rank_
```
You will still need at least once to use `importFrom` for `%>%`. 

For both options you will also need to update the DESCRIPTION file to add, the package(s) you want to import in the `imports` sections, for example:
```
Imports: 
    dplyr,
    scoringutils,
    stats
```